src/components/dialog/OverlayFadingBackground.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {useRef, useEffect, useCallback, useMemo} from 'react';
+import View from '../view';
+import {Animated} from 'react-native';
+
+interface Props {
+  dialogVisibility: boolean;
+  modalVisibility: boolean;
+  overlayBackgroundColor: string;
+}
+
+export default ({
+  dialogVisibility,
+  modalVisibility,
+  overlayBackgroundColor
+}: Props) => {
+  const fadeAnimation = useRef(new Animated.Value(0)).current;
+
+  useEffect(() => {
+    if (!dialogVisibility) {
+      animateFading(0);
+    }
+  }, [dialogVisibility]);
+
+  useEffect(() => {
+    if (modalVisibility) {
+      animateFading(1);
+    }
+  }, [modalVisibility]);
```

Comment:
The hooks plugin says you are missing `animateFading`

Reply:
fixed.